### PR TITLE
Suggest image name convention

### PR DESCRIPTION
**Pull request by jonhawkes**
Signed-off-by: Jon Hawkes <jon.hawkes@uk.ibm.com>

### CODE REVIEW

**Comment by pilhuhn** [comment]
That is a good start. I wonder if we should get rid of the per-implementation of "you can reach the service like... " and rather make sure it is the same all over -- especially when inside a container. This will make it easier going forward to deploy into Istio, as the deployment config is then the same for all impl except for the `-<impl>` suffix on the image name

File: README.md [hunk 1]
```diff
@@ -18,17 +14,23 @@ This is the serviceA component of the microprofile service mesh sample.
 
 The pom is designed to contain application server profiles with which you can test and run the service. Currently the *liberty* profile is provided.
 
-## Open Liberty
+You may want to deploy multiple versions of serviceA into the mesh so it's important to consider the uniqueness of image names when building the docker containers. A simple strategy would be to include the profile in the image name. For example, a suitable image name for a Liberty container could be &lt;docker id&gt;/serviceb-liberty. Using an image name of this form will allow you to use the script supplied in the [microprofile-service-mesh](https://github.com/eclipse/microprofile-service-mesh) repository to deploy your services into a cluster.
```
Comment (pilhuhn) [inline README.md]: The other option is to have the profile as tag as in  `serviceb:thorntail `and `serviceb:liberty`. Not sure which way is better - especially when one wants in the future also have a real version tag.
@knrc wdyt?

Comment (knrc) [reply README.md]: @pilhuhn Did you really mean me?

**Comment by jonhawkes** [comment]
Superceded

**Pull request closed without merging**